tools/scenario-tools/lib/tasks.js

```diff
@@ -90,6 +90,11 @@ async function processTask (newTask, taskspath = TASKS_FILE_PATH,
   progresspath = PROGRESS_FILE_PATH, log = logger) {
   const { name, tasks } = loadYamlFile(taskspath)
 
+  if (tasks === undefined) {
+    logger.error('A scenario cannot be found! Has one been provisioned with `simulator scenario launch ...`?')
+    process.exit(1)
```

Comment:
We should `throw new Error(<message>)` here and not log an error and use `process.exit` that will keep the responsibility of exiting the process in one place in `bin/process_task.js`